package.json

```diff
@@ -16,7 +16,9 @@
       "packages/related-table",
       "packages/table",
       "packages/components",
-      "packages/react-components"
+      "packages/source-iottwinmaker",
+      "packages/react-components",
+      "packages/scene-composer"
```

Comment:
should scene composer component come before the other component packages? The list of packages is sensitive as it determines the build order

Reply:
I don't see any build issues... the composer also has source-iottwinmaker as a dependency, so feel it could be the last

Reply:
Shouldn't `scene-composer` be a dependency of `react-components`? and why does `react-components` rly on `source-twnmaker`?

Reply:
I believe `source-twinmaker` is used in the video player test to create the video data object.
how do we want to export composer from react-components? i can do it as followup. like this in its index file?
```
export * from '@iot-app-kit/scene-composer';
```